apps/candle-service/.env.example

```
@@ -0,0 +1,6 @@
+PORT=50054
+KAFKA_BROKERS=localhost:19092,localhost:19093,localhost:19094
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The KAFKA_BROKERS key should go before the PORT key

(UnorderedKey)